src/index.css

```diff
@@ -2,11 +2,10 @@
   font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
   line-height: 1.5;
   font-weight: 400;
-
+  /* stylelint-disable csstree/validator */
```

Comment:
It's not a good practise to disable linters, kindly resolve the issue that the linter is pointing out. Thank you. 🙏